### ISSUE TITLE
Inappropriate Question: Any alternatives on Linux?

### ISSUE THREAD

**Issue by rzcat** [Pascal-Krenckel/CompressedFileViewer#1]
I like this plugin. Thank you very much.

But npp is for Windows only, there is no Linux version.
I'm looking for an alternative on Linux, which can view compressed text file directly, without it gets decompressed beforehand manually.
Something like the GUI version of zcat and xzcat.

Any suggestions?

**Comment by Pascal-Krenckel** [comment]
No, I specifically programmed this plugin to view linux log files on my server with Notepad++ and FileZilla.
